.circleci/config.yml

```diff
@@ -23,23 +18,13 @@ jobs:
           command: |
             echo ${GHCR_PAT} | docker login https://ghcr.io -u ${GHCR_USERNAME} --password-stdin
 
-      - run:
-          name: Update docker and docker compose
-          command: |
-            sudo apt-get purge docker-ce containerd.io
-            curl -fsSL https://get.docker.com -o get-docker.sh
-            sudo sh get-docker.sh
-            sudo curl -L "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-            sudo chmod +x /usr/local/bin/docker-compose
-
       - run:
           name: checkout scalar-samples repository as it contains the script to spin up scalardl environment and easier for us to maintain the CI
           command: |
             git init
             git pull https://${GHCR_USERNAME}:${GHCR_PAT}@github.com/scalar-labs/scalar-samples.git
```

Comment:
```suggestion
            git pull https://${GHCR_USERNAME}:${GHCR_PAT}@github.com/scalar-labs/scalardl-samples.git
```

Reply:
Thank you for the suggested change. Maybe CI fails due to this. But I'm curious about why `git pull` follows the change of the repo's name... In my environment, it seems to redirect `scalardl-samples` automatically.